### PR TITLE
Fix incorrect spelling of German translation

### PR DESCRIPTION
**Pull request by CennoxX**
Hi,
this pull request fixes some typos in the German translating.

### CODE REVIEW

File: translations/DE.ini [hunk 1]
```diff
@@ -2141,10 +2141,10 @@ SCRIPT_EXIT_DESC="Schließt alle Sitzungen und beendet das Programm."
 SCRIPT_OPEN_DESC="Verbindet zum Server."
 SCRIPT_CLOSE_DESC="Schließt die Sitzung."
 SCRIPT_SESSION_DESC="Zeigt alle verbundenen Sitzungen oder wählt die aktive Sitzung aus."
-SCRIPT_PWD_DESC="Zeigt den Name das entfernte Arbeitsverzeichnisses an."
+SCRIPT_PWD_DESC="Zeigt den Namen des entfernten Arbeitsverzeichnisses an."
 SCRIPT_CD_DESC="Wechselt das entfernte Arbeitsverzeichnis."
 SCRIPT_LS_DESC="Zeigt den Inhalt des entfernten Ordners an."
-SCRIPT_LPWD_DESC="Zeigt den Name des lokalen Arbeitsverzeichnisses an."
+SCRIPT_LPWD_DESC="Zeigt den Namen<< des lokalen Arbeitsverzeichnisses an."
```
Comment (martinprikryl) [inline translations/DE.ini:2147]: Thanks for the corrections – The `<<` here looks like a typo. Is it?

Comment (CennoxX) [reply translations/DE.ini:2147]: yes it is

Comment (CennoxX) [reply translations/DE.ini]: i corrected it.

**Comment by martinprikryl** [comment]
Thanks. Please submit your changes to SetupAppRunningError and UninstallAppRunningError to https://github.com/jrsoftware/issrc/blob/main/Files/Languages/German.isl

**Pull request merged**